src/MycilaTrafficLight.cpp

```diff
@@ -54,6 +54,29 @@ void Mycila::TrafficLight::begin(const int8_t greenPin, const int8_t yellowPin,
   }
 }
 
+#ifdef MYCILA_NEOPIXEL_SUPPORT
+void Mycila::TrafficLight::begin(const int16_t neoPin, const uint16_t neoCount, const neoPixelType neoType) {
```

Comment:
`int8_t neoPin` would be enough for a pin ?

Reply:
Sure, as behind the scene I'm calling Adafruit lib, I just let same type they made